### PR TITLE
Add "alpine3.7" variant of 2.4 & 2.5-rc

### PR DESCRIPTION
**Pull request by ojab**

### CODE REVIEW

**Comment by ojab** [comment]
Not sure if `defaultAlpineVersion` in `generate-stackbrew-library.sh` should be changed to `3.7`, left it at `3.6`.

**Comment by tianon** [comment]
Yeah, we should definitely update `defaultAlpineVersion`, and add `[2.4]='3.6'` in the array below it. :+1: (We don't worry about updating `-alpine` for RCs, but for GA releases we need to come up with a better process, so for now we leave it alone.)

Thanks for taking this on and making a PR!

**Comment by ojab** [comment]
Pushed the updated version.

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -12,11 +12,12 @@ declare -A debianSuites=(
 	[2.3]='jessie'
 	[2.4]='jessie'
 )
-defaultAlpineVersion='3.6'
+defaultAlpineVersion='3.7'
 declare -A alpineVersions=(
 	[2.2]='3.4'
 	[2.3]='3.4'
 	[2.4]='3.4'
+	[2.4]='3.6'
```
Comment (tianon) [inline generate-stackbrew-library.sh]: Doh, sorry, I didn't realize this already had an entry for 2.4, I told you wrong. 😞

**Comment by tianon** [comment]
LGTM, thanks :+1:

**Pull request merged**